cmd/xamarin.go

```diff
@@ -76,6 +76,38 @@ func archivableSolutionConfigNames(projectsByID map[string]project.Model) []stri
 	return archivableSolutionConfigNames
 }
 
+// findSolution scans the directory for Xamarin.Solution file first
+// If can't find any, ask the user to drag-and-drop the file
+func findSolution() (string, error) {
+	var solutionPth string
+	solPaths, err := scanForProjectFiles(xamarinProjectType)
+	if err != nil {
+		log.Printf("Failed: %s", err)
+		fmt.Println()
```

Comment:
Don't mix logging and pure `fmt` calls. Their output stream can be set up differently, which can cause impaired logging.

Reply:
Currently our log package support Println() method.